Products/CMFPlone/utils.py

```diff
@@ -505,21 +505,27 @@ def safe_unicode(value, encoding='utf-8'):
     return value
 
 
-def safe_encode(value, encoding='utf-8'):
-    """Convert unicode to the specified encoding.
+safe_unicode = safe_text
```

Comment:
why not include a deprecation warning on this?

Reply:
Because safe_unicode is used widely and that would then require a lot of work to get rid of the warnings. I don't see a problem with continuing to support the old name.